Library/Formula/sgfutils.rb

```diff
@@ -0,0 +1,68 @@
+class Sgfutils < Formula
+  desc "collection of command-line utilities to work with SGF files"
+  homepage "http://homepages.cwi.nl/~aeb/go/sgfutils/html/sgfutils.html"
+  url "http://homepages.cwi.nl/~aeb/go/sgfutils/sgfutils-0.22.tgz"
+  sha256 "4ad83be91ac3a52dd8bd0766bfe403499430da96e680c5aea2f5381ba0b34923"
+
+  depends_on "openssl" => :build
+
+  # Makefile tweaks for OS X. The corresponding patch was sent to the
+  # main author of sgfutils on 2016-03-17 for potential upstream
+  # inclusion.
```

Comment:
Alright. We might sit on this for a few days to see if you get a response if there's no public tracker.